Documentation/ExtensionArchitecture/Extbase/Reference/Validation/Validators/Index.rst

```diff
@@ -0,0 +1,624 @@
+:navigation-title: List of Validators
+
+..  include:: /Includes.rst.txt
+..  _extbase-validator:
+
+=======================================
+Built-in Validators provided by Extbase
+=======================================
+
+This document lists all built-in Extbase validators available in TYPO3 13,
```

Comment:
Better skip the version unless you want to update it with every major version. Or do we have a generated placeholder for this?